### ISSUE TITLE
[QUESTION] What is the lifetime of objects that I build in the world?

### ISSUE THREAD

**Issue by idgas01** [VanillaPlusPlus/VPP-Admin-Tools#59]
Let's say that I find a pond in the world and there are no zombies around.  I want to build a house and garage type building next to the pond.  So I open up the VPPAdminTools and I build them.  How long will this house and building last for?  Will it be permanent?  I hope so.  I have no idea why, but some video tutorial guy made a video on how to build objects on your server.  And he had you doing it in offline mode with BaseBuilderPlus, and then copying some information to the init file.  If VPPAdminTools can do the same thing in online mode and the objects don't eventually despawn, I think that I will use VPP instead.  But I need to be sure first.  So that's why I am asking.

**Comment by GravityWolfNotAmused** [comment]
The houses placed with our Object Manager utility does not have a lifetime. They are completely persistent and get spawned back when the server restarts.

**Comment by idgas01** [comment]
Do all objects placed with the Object Manager completely persistent, or only some of them?  If only some of them, how do I know what is persistent and what isn't?  And thanks for replying.

**Comment by GravityWolfNotAmused** [comment]
Sorry, let me rephrase what I said, all scene items placed with the object manager are persistent regardless of type.
